man/systemd.net-naming-scheme.xml

```diff
@@ -534,6 +534,9 @@
 
```

Comment:
commit message: what is phandle? typo?

Reply:
"phandles" is the mechanism used for internal references in a Device Tree: https://devicetree-specification.readthedocs.io/en/latest/chapter2-devicetree-basics.html#phandle (basically whenever you have a non-alias DT property referencing another node using `&label`, you have a phandle reference)

Reply:
Thank you!